### PR TITLE
Add comment for dygraph api

### PR DESCRIPTION
**Pull request by phlrain**
Add comment for dygraph api

### CODE REVIEW

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py [hunk 1]
```diff
@@ -73,6 +119,39 @@ def step(self):
 
 
 class NaturalExpDecay(LearningRateDecay):
+    """
+    Applies natural exponential decay to the initial learning rate.
+
+    >>> if not staircase:
```
Comment (JiabinYang) [inline python/paddle/fluid/dygraph/learning_rate_scheduler.py]: `>>>`, this kind of grammar may not display properly on web?

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py [hunk 2]
```diff
@@ -99,6 +178,43 @@ def step(self):
...
 class ExponentialDecay(LearningRateDecay):
+    """
+    Applies exponential decay to the learning rate.
+
+    When training a model, it is often recommended to lower the learning rate as the
+    training progresses. By using this function, the learning rate will be decayed by
+    'decay_rate' every 'decay_steps' steps.
+
+    >>> if staircase == True:
+    >>>     decayed_learning_rate = learning_rate * decay_rate ^ floor(global_step / decay_steps)
```
Comment (JiabinYang) [inline python/paddle/fluid/dygraph/learning_rate_scheduler.py]: same to above

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py [hunk 3]
```diff
@@ -125,6 +241,43 @@ def step(self):
...
 
 class InverseTimeDecay(LearningRateDecay):
+    """
+    Applies inverse time decay to the initial learning rate.
+
+    When training a model, it is often recommended to lower the learning rate as the
+    training progresses. By using this function, an inverse decay function will be
+    applied to the initial learning rate.
+
+    >>> if staircase == True:
```
Comment (JiabinYang) [inline python/paddle/fluid/dygraph/learning_rate_scheduler.py:255]: same

File: python/paddle/fluid/dygraph/nn.py [hunk 4]
```diff
@@ -169,22 +179,26 @@ def __init__(self,
         #  kernel fixed https://github.com/PaddlePaddle/Paddle/issues/17275
         self._l_type = 'conv2d'
 
-        if groups is None:
-            num_filter_channels = num_channels
+    def build_once(self, input):
```
Comment (JiabinYang) [inline python/paddle/fluid/dygraph/nn.py]: this may need to be changed to `_build_once`

**Review by JiabinYang — COMMENTED** [review]
some comments and plz let @junjun315 review this too

**Review by JiabinYang — COMMENTED** [review]
LGTM

**Review by junjun315 — APPROVED** [review]
LGTM

**Pull request merged**